code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -68,6 +68,20 @@
 	skirts["jumper dress"] = /obj/item/clothing/under/skirt/jumper_highcut
 	gear_tweaks += new /datum/gear_tweak/path(skirts)
 
+/datum/gear/uniform/suitskirt
+	display_name = "Skirtsuit (recolourable)"
```

Comment:
```suggestion
	display_name = "skirtsuit (recolourable)"
```

Reply:
huh, never realised none of the display names are not capitalised.